examples/qml_features/rust/src/signals.rs

```diff
@@ -25,7 +25,10 @@ pub mod ffi {
     // ANCHOR: book_signals_struct
     #[cxx_qt::qobject]
     #[derive(Default)]
-    pub struct RustSignals;
+    pub struct RustSignals {
```

Comment:
hmm this example was supposed to be showing signals not properties :thinking: 

Maybe you can reuse the properties.rs example for the unit test. Eg just have an additional test in `tst_properties.qml` which creates a `RustProperties` then call `disconnect` multiple times and see if the properties like `connected` change or something :thinking:

Reply:
Oh I didn't realize that this file would be embedded in the docs. In that case I shall remove the property there, even though the test technically tested behaviour of signals.